backend/config/storage.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+type StorageConfig struct {
+	Path string `yaml:"path"`
+}
+
+func validateStorageConfig(cfg *Config) error {
+	if cfg.Storage == nil {
+		cfg.Storage = &StorageConfig{
+			Path: DefaultStoragePath,
+		}
+	}
+
+	if len(cfg.Storage.Path) == 0 {
+		cfg.Storage.Path = DefaultStoragePath
```

Comment:
Made a note to later utilize [the XDG spec](https://specifications.freedesktop.org/basedir-spec/latest/index.html#introduction), this is good for now ✅